api-reference/v1.0/api/device_post_registeredowners.md

```diff
@@ -17,11 +17,10 @@ POST /devices/{id}/registeredOwners
 | Authorization  | string  | Bearer <token>. Required. |
 
 ## Request body
-In the request body, supply a JSON representation of [directoryObject](../resources/directoryobject.md) object.
-
+In the request body, supply a JSON representation of a [user](../resources/user.md) to be added.
```

Comment:
Same comment here as above -- unless you can actually supply a user object, aren't we really supplying a reference?